### ISSUE TITLE
Dind Tests Failure

### ISSUE THREAD

**Issue by Nekroze** [Nekroze/dab#133]
### Describe the bug

None.

### To Reproduce

1. Run `./scripts/test.sh`

### Expected behavior

Should pass.

### Example

None.

### `dab version` output

```
Dab Version: 6511ddaf1a3f6dc6023fce098fb4d6c382299c69
Server Version: 18.06.1-ce
Runtimes: runc
Kernel Version: 4.14.76
Operating System: NixOS 18.09.932.09195057114 (Jellyfish)
Architecture: x86_64
```

### Additional context

Might be related to dind performance.

**Comment by Nekroze** [comment]
I believe this relates to https://github.com/docker/compose/releases/tag/1.23.0 update adding a slug to the end of container names with is to be resolved in #140

**Comment by Nekroze** [comment]
Closing as a duplicate of #348
